services/core-api/app/api/projects/project_summary/models/project_summary.py

```diff
@@ -929,6 +938,48 @@ def validate_project_summary(cls, data, is_historic):
 
         return errors_found
 
+    def send_status_notification(self, prev_status, mine):
```

Comment:
just moved this from the resource file, basically, and then added the bit with `noun` and final `elif` if in there